Classes/Ttree/Scheduler/Service/TaskService.php

```diff
@@ -80,17 +80,28 @@ public static function getAllDynamicTaskImplementations(ObjectManagerInterface $
     {
         $tasks = [];
         /** @var ReflectionService $reflectionService */
-        $reflectionService = $objectManager->get('TYPO3\Flow\Reflection\ReflectionService');
+        $reflectionService = $objectManager->get(ReflectionService::class);
         foreach (self::getAllTaskImplementations($objectManager) as $className) {
```

Comment:
If we improve the getAllTaskImplementations method, maybe we can remove all the call to the reflexion service in production in this loop (86, 91, 98) ?